swift3/controllers/cors.py

```diff
@@ -44,9 +44,8 @@ def match(pattern, value):
 
 
 def get_cors(app, req, method, origin):
-    resp = req._get_response(app, 'HEAD',
-                             req.container_name, "")
-    body = resp.sysmeta_headers.get(BUCKET_CORS_HEADER)
+    sysmeta = req.get_container_info(app).get('sysmeta', {})
+    body = sysmeta.get(BUCKET_CORS_HEADER) or sysmeta.get('swift3-cors')
```

Comment:
The prefix is always removed by `swift.common.request_helpers.strip_sys_meta_prefix`. The `BUCKET_CORS_HEADER` key will never work.